### PR TITLE
Imjoy screenshot, version doc

### PR DESCRIPTION
**Pull request by thewtex**

### CODE REVIEW

File: doc/content/docs/imjoy.md [hunk 1]
```diff
@@ -5,16 +5,24 @@ title: ImJoy Plugin
...
 ```
 https://kitware.github.io/itk-vtk-viewer/app/
 ```
 
 Note that the link can also be used directly.
 
+To install a specific version, e.g. version `10.8.0`, use the URI:
+
+```
+https://unpkg.com/itk-vtk-viewer@10.8.0/dist/index.html
```
Comment (oeway) [inline doc/content/docs/imjoy.md:23]: Didn't know unpkg serves static site, this is a really cool feature for reproducibility. I was using netlify PR previews, but it is limited for build time for free account.

File: dist/index.html [hunk 2]
```diff
@@ -60,7 +60,7 @@
         license: '3-Clause BSD License',
         labels: ['visualization', 'itk', 'vtk', 'image', 'mesh', 'geometry', 'point set', '2D', '3D'],
         icon: 'https://kitware.github.io/itk-vtk-viewer/app/favicon-32x32.png',
-        cover: ['https://kitware.github.io/itk-vtk-viewer/docs/howToUse.jpg'],
+        cover: ['https://kitware.github.io/itk-vtk-viewer/docs/howToUse.jpg', 'https://kitware.github.io/itk-vtk-viewer/docs/imjoy.png'],
```
Comment (oeway) [inline dist/index.html:63]: Good!

**Review by oeway — APPROVED** [review]

File: dist/index.html (hunk 2 above)
Comment (oeway) [reply dist/index.html:63]: It seems we have some display issue in imjoy.io: https://github.com/imjoy-team/ImJoy/issues/394

File: doc/content/docs/imjoy.md (hunk 1 above)
Comment (thewtex) [reply doc/content/docs/imjoy.md:23]: The unpkg CDN is a nice, automatic bonus that comes when publishing to NPM -- yes, specific versions should help with reproducibility :-)

**Pull request merged**